README.md

```diff
@@ -35,6 +35,9 @@ A list of environment variables as build-args
         buildargs: PACKAGE_NAME,VCS_REF
 ```
 
+### `global`
+
+When true will make the image available to everyone in your team. Default `false`.
```

Comment:
Can you mention that you need to be an admin user to use this?

Reply:
@teresaromero could you take care of ^^?